### ISSUE TITLE
install sysfonts error

### ISSUE THREAD

**Issue by choumeijiao** [yixuan/sysfonts#15]
* installing *source* package ‘sysfonts’ ...
** 成功将‘sysfonts’程序包解包并MD5和检查
** using staged installation
checking for gcc... x86_64-conda_cos6-linux-gnu-cc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether x86_64-conda_cos6-linux-gnu-cc accepts -g... yes
checking for x86_64-conda_cos6-linux-gnu-cc option to accept ISO C89... none needed
checking for pkg-config... /usr/bin/pkg-config
checking whether pkg-config could find zlib... yes
checking whether pkg-config could find libpng... yes
checking whether pkg-config could find freetype2... yes
configure: creating ./config.status
config.status: creating src/Makevars
** libs
x86_64-conda_cos6-linux-gnu-cc -I"/TJPROJ5/NEWBORN/software/anaconda3/lib/R/include" -DNDEBUG -I/usr/include/freetype2   -I/usr/include/libpng12      -DNDEBUG -D_FORTIFY_SOURCE=2 -O2  -I/TJPROJ5/NEWBORN/software/anaconda3/include -Wl,-rpath-link,/TJPROJ5/NEWBORN/software/anaconda3/lib  -fpic  -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -I/TJPROJ5/NEWBORN/software/anaconda3/include -fdebug-prefix-map=/tmp/build/80754af9/r-base_1570124924484/work=/usr/local/src/conda/r-base-3.6.1 -fdebug-prefix-map=/TJPROJ5/NEWBORN/software/anaconda3=/usr/local/src/conda-prefix  -c font_name.c -o font_name.o
In file included from font_name.c:1:0:
sysfonts.h:8:10: fatal error: ft2build.h: No such file or directory
 #include <ft2build.h>
          ^~~~~~~~~~~~
compilation terminated.
make: *** [/TJPROJ5/NEWBORN/software/anaconda3/lib/R/etc/Makeconf:168: font_name.o] Error 1
ERROR: compilation failed for package ‘sysfonts’
* removing ‘/TJPROJ5/NEWBORN/software/anaconda3/lib/R/library/sysfonts’
ERROR: dependency ‘sysfonts’ is not available for package ‘showtextdb’
* removing ‘/TJPROJ5/NEWBORN/software/anaconda3/lib/R/library/showtextdb’
ERROR: dependencies ‘sysfonts’, ‘showtextdb’ are not available for package ‘showtext’
* removing ‘/TJPROJ5/NEWBORN/software/anaconda3/lib/R/library/showtext’

下载的程序包在
	‘/tmp/RtmpJHl9TF/downloaded_packages’里
更新'.Library'里的HTML程序包列表
Making 'packages.html' ... 做完了。
Warning messages:
1: In install.packages("showtext", repos = "https://mirrors.ustc.edu.cn/CRAN/") :
  安装程序包‘sysfonts’时退出狀態的值不是0
2: In install.packages("showtext", repos = "https://mirrors.ustc.edu.cn/CRAN/") :
  安装程序包‘showtextdb’时退出狀態的值不是0
3: In install.packages("showtext", repos = "https://mirrors.ustc.edu.cn/CRAN/") :
  安装程序包‘showtext’时退出狀態的值不是0

**Comment by yixuan** [comment]
Can you check the folder `/usr/include/freetype2` to see if the file `ft2build.h` exists?

**Comment by choumeijiao** [comment]
> Can you check the folder `/usr/include/freetype2` to see if the file `ft2build.h` exists?

Does not exist.Please tell me what should I do

**Comment by choumeijiao** [comment]
It's here,/usr/include/ft2build.h.But i'm not root.

**Comment by yixuan** [comment]
That looks weird. Can you run `pkg-config freetype2 --cflags` in bash to see the result? I suspect that the FreeType library was not properly installed on the machine.

**Comment by choumeijiao** [comment]
pkg-config freetype2 --cflags
-I/usr/include/freetype2
Sad!What's wrong?

**Comment by yixuan** [comment]
Yeah, that's also my guess... Maybe the `libfreetype6-dev` package (assuming a Ubuntu OS) has not been properly installed. I suggest contacting the administrator to report this issue and ask them to install the missing library.
